regulations/static/regulations/css/less/module/drawer-content.less

```diff
@@ -90,8 +90,8 @@ This contains all of the styles specific to the TOC
 
   a:link,
   a:visited {
-    color: @black;
-    .border-bottom-light (@width: 1px);
+    color: @navigation_gray;
+    .border-bottom-light (@width: 0px);
```

Comment:
I don't think the border is specific to ATF -- we want to keep it for all CFR viewers, don't we? There are often _lots_ of sections in between subpart dividers. Should the absence of a border be specific to the _preamble_ (i.e. not EPA specific)?